src/components/profile-sidebar/profile-sidebar.jsx

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import { useState } from "react";
+import {
+  Box,
+  List,
+  ListItemButton,
+  ListItemIcon,
+  ListItemText,
+} from "@mui/material";
+import styles from "./profile-sidebar.module.css";
+import { RightChevron } from "../icons";
+
+export function SideBar() {
+  const [selected, setSelected] = useState(false);
+
+  const sidebarSections = [
+    "Personal Information",
+    "Refer and Earn",
+    "My Orders",
+    "My Wishlist",
+    "Mt Reviews",
+    "My Address Book",
+    "My Saved Cards",
+  ];
+
+  const handleClick = (index) => {
```

Comment:
Better name for the function, what was clicked ? Does this handle anything that's clicked ?